src/components/Banner/Banner.tsx

```diff
@@ -1,11 +1,18 @@
+import { Link } from 'react-router-dom';
 import { Title } from '@mantine/core';
 import classes from './Banner.module.css';
+import { CURRENT_PUBLISHED_WEEK } from '../../settings';
 
 export default function Banner() {
   return (
-    <div className={classes.banner}>
```

Comment:
I made the banner a link to the current week because I kept clicking on it thinking it was!